content/intuitionistic-logic/introduction/bhk-interpretation.tex

```diff
@@ -33,7 +33,7 @@
 \item A construction of $!A_1 \lor !A_2$ is a pair $\tuple{s, M}$
   where $s$ is~$1$ and $M$ is a construction of~$!A_1$, or $s$ is~$2$
   and $M$ is a construction of~$!A_2$.
-\item A construction of $!A \lif !B$ is a function that converts a
+\item A construction of $!A \lif !B$ is a computable function that converts a
```

Comment:
I believe it's better to keep the notion a bit vague here and leave it at "function".